src/named.rs

```diff
@@ -1915,6 +1915,180 @@ impl NamedChain {
 
         Some(addr)
     }
+
+    /// Returns the address of the ENScribe contract, if it exists.
```

Comment:
can we add some additional context about what ENScribe is, like an url or a reference contract